include/net/fota_download.h

```diff
@@ -112,6 +118,36 @@ typedef void (*fota_download_callback_t)(const struct fota_download_evt *evt);
  */
 int fota_download_init(fota_download_callback_t client_callback);
 
+/**@brief Start downloading the given file from the given host. Validate that the
+ * file type matches the expected type before starting the installation.
+ *
+ * When the download is complete, the secondary slot of MCUboot is tagged as having
+ * valid firmware inside it. The completion is reported through an event.
+ *
+ * URI parameters (host and file) are not copied, so pointers must stay valid
+ * until download is finished.
+ *
+ * @param host Name of host to start downloading from. Can include scheme
+ *             and port number, for example https://google.com:443
+ * @param file Path to the file you wish to download. See fota_download_any()
+ *             for details on expected format.
+ * @param sec_tag_list Security tags that you want to use with HTTPS. Pass NULL to disable TLS.
+ * @param sec_tag_count Number of TLS security tags in list. If set to 0, TLS is disabled.
```

Comment:
```suggestion
 * @param sec_tag_count Number of TLS security tags in list.  Pass 0 to disable TLS. 
```
To stay consistent with previous line.